examples/slurm/submit_multicpu.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -l
+
+#SBATCH --job-name=multicpu
+#SBATCH --nodes=2                       # number of Nodes
+#SBATCH --ntasks-per-node=1             # number of MP tasks
+#SBATCH --exclusive
+#SBATCH --output=output/torch_%A.log
```

Comment:
To be more consistent with the other scripts we should use the following file paths for output and errors:
```
#SBATCH --output=O-%x.%j
#SBATCH --error=E-%x.%j
```